packages/in_app_purchase/lib/src/store_kit_wrappers/sk_payment_queue_wrapper.dart

```diff
@@ -137,6 +137,17 @@ class SKPaymentQueueWrapper {
         applicationUserName);
   }
 
+  /// Present Code Redemption Sheet
+  ///
+  /// Use this to allow Users to enter and redeem Codes
+  ///
+  /// This method triggers [`-[SKPayment
+  /// presentCodeRedemptionSheet]`](https://developer.apple.com/documentation/storekit/skpaymentqueue/3566726-presentcoderedemptionsheet?language=objc)
+  Future<void> presentCodeRedemptionSheet() async {
```

Comment:
looks like we missed a dart unit test for this method?

Reply:
Added